src/DynamoCore/Models/DynamoModel.cs

```diff
@@ -2277,9 +2277,11 @@ private void CheckForXMLDummyNodes(WorkspaceModel workspace)
                   Resources.UnresolvedNodesWarningTitle,
                   Resources.UnresolvedNodesWarningShortMessage,
                   Resources.UnresolvedNodesWarningMessage);
-
+                if (!IsTestMode)
+                {
+                    DisplayXmlDummyNodeWarning();
```

Comment:
We have been hidding the use/logic related to `IsTestMode` on lower level by passing it in. I feel like this maybe a general issue, do you think we should put it in places like `RequestTaskDialog` more?

Reply:
I prefer the explicitness of this - if I raise an event I want want it to get raised. IsTestMode seems to get used a bunch directly in DynamoModel.

Reply:
I wouldn't mind moving it into the display XmlDummyNodeWarning method though if thats preferable?

Reply:
@mjkkirschner After thinking about this, let's keep it in the outer scopt since `Logging.Analytics` should be skipped for test mode as well